packages/amplify-graphql-transformer-core/src/transformation/transform.ts

```diff
@@ -393,6 +393,7 @@ export class GraphQLTransform {
         external: true,
       },
     });
+    // Remove these when moving override up to amplify-category-api level
```

Comment:
It would be possible to remove these references to `stateManager` now, but it would require a larger effort. Because we plan to move override functionality up to api-category level (which will retain access to `stateManager`) it is easier to leave this for now.